build.py

```diff
@@ -1113,8 +1113,36 @@ def linked_names(dim: str, fname: str, cell: str) -> str:
     return ", ".join(out)
 
 
-de_rham_3d = []
-de_rham_2d = []
+def de_rham_row(family: typing.Dict[str, typing.Dict[str, typing.List[str]]],
+                fname: str, cell: str, diff_form_deg: typing.List[str]) -> str:
+    """Create HTML string of FEs.
+
+    Args:
+        family: Elements
+        fname: Filename
+        cell: Reference cell
+        diff_form_deg: List of differential form degrees
+
+    Returns:
+        HTML string of FEs
+    """
+    row = ""
+    if all(i in family[cell] for i in diff_form_deg):
+        row += "<tr>"
+        row += f"<td>{linked_names(str(len(diff_form_deg)), fname, cell)}</td>"
+        for o in diff_form_deg:
+            row += f"<td><a href='/elements/{family[cell][o][1]}'"
+            row += " style='text-decoration:none'>"
+            row += f"{family[cell][o][0]}</a></td>"
+            if o != "d":
+                row += "<td>&nbsp;</td>"
+        row += "</tr>"
+    return row
+
+
+de_rham_3d: typing.List[str] = []
+de_rham_2d: typing.List[str] = []
+de_rham_2D: typing.List[str] = []
```

Comment:
can we make the variable name here more distinct from the one above? Having the only difference be uppercase vs lowercase can be confusing